flask/test/data/box.py

```diff
@@ -0,0 +1,21 @@
+from datetime import datetime
+
+import pytest
+from boxwise_flask.models.box import Box
+
+
+@pytest.fixture()
+def default_box(default_box_state, default_location):
+    mock_box = {
```

Comment:
box always needs a product in it

Reply:
why is product nullable in the database?

Reply:
Yeah we should fix that in the database.